### DISCUSSION TITLE
Calibre-Web in Organizr iFrame

### DISCUSSION THREAD

**Discussion by Sithlord82** [htpcBeginner/docker-traefik#241] [Q&A]
Having issues getting the above to work. I have it working perfectly at https://calibre-web.mydomain.com but when trying to access it in mydomain.com iFrame it doesnt load. What would you need to see shared to help me figure out how to bypass that? I've seen a fair bit to do with x-forwawrd headers but dont yet know enough to be able to configure it fully

**Comment by Sithlord82** [comment]
Anyone come across a fix for this yet?

**Comment by uasdj25** [comment]
Are you using Google Oauth? Authelia?

I'd try to disable those first if so, and then see if it works in Organizr.  If you're not using 2FA, then you can reach out to the Organizr folks via their Discord channel.  They can help you out there.  They've helped me out a couple of times, really good group of people.
